mate-panel/panel-run-dialog.c

```diff
@@ -1748,10 +1724,19 @@ panel_run_dialog_setup_entry (PanelRunDialog *dialog,
 	GtkWidget             *entry;
 
 	dialog->combobox = PANEL_GTK_BUILDER_GET (gui, "comboboxentry");
+	dialog->possible_executables = fill_possible_executables ();
+	dialog->dir_hash = g_hash_table_new_full (g_str_hash,
+							  g_str_equal,
+							  g_free, NULL);
```

Comment:
Indents

Reply:
Update indent

Reply:
See how it looks in browser now :/
https://github.com/mate-desktop/mate-panel/pull/1219/commits/b3a27410cb1d462fa3856c557dd29d74446038e0

Reply:
Update again

Reply:
Thx :)